.github/workflows/rust.yml

```
@@ -268,11 +268,11 @@ jobs:
 
       - uses: actions-rs/toolchain@v1
         with:
-          toolchain: 1.56.1
+          toolchain: 1.60
```

This is a floating point number for yaml... It installed Rust 1.6 :joy: 
```suggestion
          toolchain: "1.60.1"
```